dist/automate.go

```diff
@@ -1,158 +0,0 @@
-package main
```

Comment:
This was throwing all kinds of build errors and I think there are easier ways to achieve this.